doc/conf.py

```diff
@@ -110,10 +110,11 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-#html_title = None
+# Don't show version in title
+html_title = "FOSHttpCache Documentation"
```

Comment:
As we’re only maintaining one major version now, I left that out for the time being. When we start having docs for version 2 as well as 1, we can always add it in.